app/views/unattended/provisioning_templates/finish/alterator_default_finish.erb

```diff
@@ -18,10 +18,11 @@ cat > /etc/puppet/puppet.conf << EOF
 <%= snippet 'puppet.conf' -%>
 EOF
 
-/usr/bin/puppet agent --config /etc/puppet/puppet.conf -o --tags no_such_tag --server <%= @host.puppetmaster %> --no-daemonize
-<%= snippet 'built' %>
+/usr/bin/puppet agent --config /etc/puppet/puppet.conf -o --tags no_such_tag --server <%= host_puppet_server %> --no-daemonize
 /sbin/chkconfig puppetd on
 
+<%= snippet 'built' %>
+
```

Comment:
This is a bit of a pet peeve of mine, but there is too much whitespace here. If you look in the rendered result, there's lots of empty lines and I think this is sufficient:
```suggestion
<%= snippet 'built' %>
```